src/bin/real-rat.rs

```diff
@@ -268,6 +252,19 @@ impl SynthLanguage for Math {
         )
     }
 
+    fn is_extractable(&self) -> bool {
```

Comment:
same as other lifted domains. I think this is actually a better name than is_in_domain FWIW